release-content/0.15/release-notes/14876_Add_Order_Independent_Transparency.md

```diff
@@ -1,4 +1,11 @@
 <!-- Add Order Independent Transparency -->
 <!-- https://github.com/bevyengine/bevy/pull/14876 -->
 
-<!-- TODO -->
+<video controls autoplay loop muted><source src="oit-demo.mp4" type="video/mp4"/></video>
+
+Before this feature, bevy only used alpha blending to render transparent meshes. We now have the option to use Order Independent Transparency when rendering transparent meshes. Instead of only sorting the mesh, this will sort every pixel that contributes to a transparent triangle. This is useful if you have a lot of transparent layers in your scene.
```

Comment:
```suggestion
Before this feature, Bevy only used alpha blending to render transparent meshes. We now have the option to use Order Independent Transparency when rendering transparent meshes. Instead of only sorting the mesh, this will sort every pixel that contributes to a transparent triangle. This is useful if you have a lot of transparent layers in your scene.
```

Reply:
```suggestion
Before this feature, bevy only used alpha blending to render transparent meshes. We now have the option to use Order Independent Transparency when rendering transparent meshes. Instead of only sorting the mesh, this will sort every pixel that contributes to a transparent triangle, giving much better results with overlapping transparent meshes. This is useful if you have a lot of transparent layers in your scene.
```

Or something to this effect.

Reply:
```suggestion
Before this feature, Bevy only used alpha blending to render transparent meshes. We now have the option to use Order Independent Transparency when rendering transparent meshes. Instead of only sorting the mesh, this will sort every pixel that contributes to a transparent triangle. This is useful if you have a lot of transparent layers in your scene.
```